pkgs/servers/unftp/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, linux-pam
+, rustPlatform
+, fetchFromGitHub
+, buildFeatures ? [ "gnu" ]
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "unFTP";
+  version = "0.14.5";
+
+  src = fetchFromGitHub {
+    owner = "bolcom";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-CPq4CH7reS5AI145l8U1nekm2MKnmoBfVqENR9QOKF4";
```

Comment:
```suggestion
    hash = "sha256-CPq4CH7reS5AI145l8U1nekm2MKnmoBfVqENR9QOKF4";
```

Reply:
Why is `hash` preferred over `sha256`? I used to use `hash`, but thought being explicit on the hash type was better.

Reply:
The "sha256-" part of the hash is already making it specific (:

Reply:
I'll provide some more details.

This comes from [nix adding support for SRI hashes](https://github.com/NixOS/nix/commit/6024dc1d97212130c19d3ff5ce6b1d102837eee6), using the `hash` argument. There are several reasons why SRI hashes are preferred to normal hashes, but I believe most of it comes down to compatibility with the rest of the ecosystem. Somewhere along the road, the old to-be-deprecated-at-some-point arguments like `sha256` also started accepting SRI hashes, creating a bit of confusion. There has been [discussions around disabling this behavior to be more consistent](https://discourse.nixos.org/t/disallow-sri-hashes-in-sha256-attributes/22976), but nothing has been done at the moment. Also see [RFC 131](https://github.com/NixOS/rfcs/pull/131) for some relevant discussions.

You'll frequently see reviewers add this suggestion to both new packages, but also version bumps for packages that still use the "legacy" hashes (with legacy in quotes because AFAIK they haven't been officially deprecated anywhere)